Dockerfile

```diff
@@ -1,38 +1,28 @@
-FROM ubuntu:14.04.4
+FROM ubuntu:14.04
 
-# Replace shell with bash so we can source files
-RUN rm /bin/sh && ln -s /bin/bash /bin/sh
-
-# Install apt package dependencies
-RUN apt-get clean all && apt-get update && \
-    apt-get -y install wget curl python make g++ git supervisor && \
```

Comment:
@nathanathan has asked I make this Dockerfile separate from the existing one, for use in dev. This one won't be used to create containers delivered to your stakeholders.